src/openloco/ui/WindowManager.cpp

```diff
@@ -790,7 +785,7 @@ namespace openloco::ui::WindowManager
     {
         close(WindowType::dropdown, 0);
 
-        FOR_ALL_WINDOWS_FROM_BACK (w)
+        for (ui::window* w = &_windows[0]; w != _windowsEnd; w++)
```

Comment:
This line appears to be wrong: rather than closing the top-most window (last window opened), the bottom-most window (first window opened) is closed.